docs/source/download.rst

```diff
@@ -1,64 +1,86 @@
-=============
-Obtain Devito
-=============
+===========================
+Download and Install Devito
+===========================
+
+There are three main approaches to installing Devito.
+For those looking for a quick “out-of-the-box” way to try Devito, we recommend
+the route of `Docker installation`_. For those looking to use Devito as part of a project
+alongside other packages or/and develop for Devito, we suggest the `pip installation`_
+or the `conda installation`_.
```

Comment:
I would separate the three into a bullet list. Docker - easiest/quickest way to try. Pip - when you are using Devito alongside other packages. Conda - when you develop for Devito.